package protoFile.hall.LeaveRoom;
message LeaveRoomRequest{
     required int32 id=1;//½ÇÉ«id
}
message LeaveRoomResponse{
     required bool result=1;
     optional string message=2;
}